src/main/java/bank/hm31/response/CurrencyApiResponse.java

```diff
@@ -0,0 +1,16 @@
+package bank.hm31.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CurrencyApiResponse {
+
+    private CurrencyApiResponseMeta meta;
+    private CurrencyApiResponseData data;
```

Comment:
```suggestion
    private Map<String, ResponseCode> data;
```
думаю так могло б спрацювати.